timemachine/fe/absolute_hydration.py

```diff
@@ -156,3 +156,125 @@ def log_prob(xs, lam):
     resample = partial(conditional_multinomial_resample, thresh=resample_thresh)
 
     return samples, lambdas, propagate, log_prob, resample
+
+
+def set_up_ahfe_for_smc_parameter_changes(
+    mol,
+    temperature=300.0,
+    pressure=1.0,
+    n_steps=1000,
+    seed=2022,
+    ff0=None,
+    ff1=None,
+    is_vacuum=False,
+):
+    """
+    Prepare a system for using SMC to generate samples under
+    different forcefields at each endstate.
+
+    Parameters
+    ----------
+    is_vacuum: bool
+        Set to True to set up the vacuum leg, using NVT.
+        Set to False to set up the solvent leg, using NPT.
+
+    Returns
+    -------
+    * reduced_potential_fxn
+    * mover at lam=0
+    * mover at lam=1
```

Comment:
Returns a single mover with lam=None -- should add comment about this.

(That intended usage is `mover.lamb = 0.0; x_next = mover.move(x)` or similar)

Reply:
* mover with lamb=None. The mover.lamb attribute must be set before use.